FSharpKoans/Asserts.fs

```diff
@@ -0,0 +1,19 @@
+namespace FSharpKoans
+
+open Expecto
+
+[<AutoOpen>]
+module Asserts =
+
+  let inline __<'a> : 'a = failwith "fill in the __"
```

Comment:
these all needed to be inline so that exceptions/test failures don't report this line, rather the line in the test file that uses these to throw.